### PR TITLE
Fix to evaluate action only once when receiving blocks

### LINKED ISSUE #843: When receiving a block, the actions are evaluated twice.
```
2020-04-07 4:00:55 PM +09:00: Try to append a block #5135 "25d3034522116de9381cf740fb48999bb9d6357f065be9cfaeb3fbd156000000"...
2020-04-07 4:00:55 PM +09:00: Trying to append block 5135: "25d3034522116de9381cf740fb48999bb9d6357f065be9cfaeb3fbd156000000"
2020-04-07 4:00:55 PM +09:00: Execute action in block 5135: "25d3034522116de9381cf740fb48999bb9d6357f065be9cfaeb3fbd156000000"
2020-04-07 4:00:55 PM +09:00: Evaluating block action in block 5135: "25d3034522116de9381cf740fb48999bb9d6357f065be9cfaeb3fbd156000000"
2020-04-07 4:00:55 PM +09:00: Try to set cache df648160f3f5a9d9aef0841f5ed48e5862046aaa
2020-04-07 4:00:55 PM +09:00: Unstaging transactions...
2020-04-07 4:00:55 PM +09:00: Block 5135: "25d3034522116de9381cf740fb48999bb9d6357f065be9cfaeb3fbd156000000" is appended.
2020-04-07 4:00:55 PM +09:00: Block #5135 "25d3034522116de9381cf740fb48999bb9d6357f065be9cfaeb3fbd156000000" was appended.
2020-04-07 4:00:55 PM +09:00: Downloaded blocks from "Unspecified/15.165.103.105:54562"@"Df648160F3F5a9D9AEf0841f5eD48e5862046aaA".
2020-04-07 4:00:55 PM +09:00: Trying to find branchpoint...
2020-04-07 4:00:55 PM +09:00: Locator's count: 23
2020-04-07 4:00:55 PM +09:00: Request taken. 0 requests are left.
...

2020-04-07 4:00:56 PM +09:00: Received 1 reply messages to 4a0ee896-5ecf-49d6-ad97-47d399724aa6 from "0xDf648160F3F5a9D9AEf0841f5eD48e5862046aaA": ["Libplanet.Net.Messages.BlockHashes"].
2020-04-07 4:00:56 PM +09:00: Received a BlockHashes message with an offset index 5135 (total 1 hashes).
2020-04-07 4:00:56 PM +09:00: Branch point is #5135 "25d3034522116de9381cf740fb48999bb9d6357f065be9cfaeb3fbd156000000".
2020-04-07 4:00:56 PM +09:00: It doesn't need to fork.
2020-04-07 4:00:56 PM +09:00: Trying to fill up previous blocks...
2020-04-07 4:00:56 PM +09:00: Render actions in block 5135: "25d3034522116de9381cf740fb48999bb9d6357f065be9cfaeb3fbd156000000"
2020-04-07 4:00:56 PM +09:00: Evaluating block action in block 5135: "25d3034522116de9381cf740fb48999bb9d6357f065be9cfaeb3fbd156000000"
2020-04-07 4:00:56 PM +09:00: Trying to broadcast blocks...
2020-04-07 4:00:56 PM +09:00: Block broadcasting complete.
2020-04-07 4:00:56 PM +09:00: Broadcasting message [Libplanet.Net.Messages.BlockHeaderMessage]
2020-04-07 4:00:56 PM +09:00: Peers to broadcast : 3
2020-04-07 4:00:56 PM +09:00: [Libplanet.Net.Messages.BlockHeaderMessage] broadcasting completed.
2020-04-07 4:00:57 PM +09:00: Request taken. 0 requests are left.
2020-04-07 4:00:57 PM +09:00: Trying to send "Libplanet.Net.Messages.BlockHeaderMessage" to "0x9ad27dD5084a5225BCa5c2e9029F1926aaC116cf.Unspecified/52.78.68.87:62034.1 (Bencodex.Types.Dictionary {
```

### PR DESCRIPTION
**Pull request by earlbread**
This fixes #843.

### CODE REVIEW

File: Libplanet/Net/Swarm.cs [hunk 1]
```diff
@@ -1803,14 +1794,11 @@ await foreach (Block<T> block in blocks)
...
 
-                        // As actions in this block should be rendered
-                        // after actions in stale blocks are unrendered,
-                        // given the `render: false` option here.
                         workspace.Append(
                             block,
                             DateTimeOffset.UtcNow,
                             evaluateActions: evaluateActions,
-                            renderActions: false
+                            renderActions: evaluateActions
```
Comment (longfin) [inline Libplanet/Net/Swarm.cs:1801]: Is it okay we render before unrender? previous comments seemed to mention it... 😕

Comment (earlbread) [reply Libplanet/Net/Swarm.cs]: You're right, I was mistaken about renders. I fixed it and added a test case for it.

File: CHANGES.md [hunk 2]
```diff
@@ -118,6 +118,8 @@ To be released.
  -  Fixed a `Swarm<T>.PreloadAsync()` method's bug that it had hung forever
     when a block failed to be fetched due to an unexpected inner exception.
     [[#839]]
+ -  Fixed a bug where actions are evaluated twice when receiving blocks.
```
Comment (dahlia) [inline CHANGES.md:121]:
```suggestion
 -  Fixed a bug where actions were evaluated twice when receiving blocks.
```

File: CHANGES.md [hunk 3]
```diff
@@ -118,6 +118,8 @@ To be released.
...
+    [[#844]]
```
Comment (dahlia) [inline CHANGES.md:122]:
```suggestion
    [[#843], [#844]]
```

URLs also should be added below.

**Pull request merged**